source/agora/node/Ledger.d

```diff
@@ -1228,6 +1238,7 @@ version (unittest)
     private final class TestLedger : ValidatingLedger
     {
         public this (KeyPair key_pair,
+            const KeyPair[] key_pairs = genesis_validator_keys,    // Genesis key pairs
```

Comment:
Why this ?

Reply:
In theory we could use any key pairs but as we always use the test GenesisBlock I could remove this.